allensdk/brain_observatory/ecephys/stimulus_analysis/__main__.py

```diff
@@ -26,22 +29,26 @@ def calculate_stimulus_metrics(args):
 
     stimulus_classes = (
                  DriftingGratings,
-                 StaticGratings,
-                 NaturalScenes,
+                 #StaticGratings,
```

Comment:
intentional?

Reply:
No, it looks like testing was being done on individual files and got checked in by accident. The only one which should be commented out is NaturalMovies.